extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -180,12 +188,13 @@ export class IframeTransport {
   sendRequest(event) {
     const frameData = IframeTransport.getFrameData(this.type_);
     dev().assert(frameData, 'Trying to send message to non-existent frame');
-    dev().assert(frameData.queue, 'Event queue is missing for ' + this.id_);
+    dev().assert(frameData.queue,
+        'Event queue is missing for ' + this.getCreativeId());
     frameData.queue.enqueue(
         /**
          * @type {!../../../src/3p-frame-messaging.IframeTransportEvent}
          */
-        ({transportId: this.id_, message: event}));
+        ({creativeId: this.getCreativeId(), message: event}));
```

Comment:
This is a breaking change, is it OK?

Reply:
I just verified that nobody is actually using iframe transport yet, so it is ok.